### PR TITLE
v2.2.0 - Add packaged/iframed static properties to capabilities (bug 1117771)

### PR DESCRIPTION
**Pull request by diox**
Following @ngokevin's suggestion in https://github.com/mozilla/fireplace/pull/900#discussion_r26507701

However, I don't know how to test it since it's a) a static property b) that depends on `window`. I'd like to avoid making a method since it doesn't need to be. Thoughts ?

### CODE REVIEW

File: core/capabilities.js [hunk 1]
```diff
@@ -44,6 +44,8 @@ define('core/capabilities', ['core/settings'], function(settings) {
         'debug': document.location.href.indexOf('dbg') >= 0,
         'debug_in_page': document.location.href.indexOf('dbginpage') >= 0,
         'console': window.console && typeof window.console.log === 'function',
+        'isPackagedApp': window.location.protocol === 'app:',
```
Comment (ngokevin) [inline core/capabilities.js]: i think capabilities.packaged and capabilities.iframed suffice

**Comment by ngokevin** [comment]
bump bower in this patch, prepend version # to commit msg, r+wc

File: core/capabilities.js (hunk 1 above)
Comment (diox) [inline core/capabilities.js]: I almost did that, but I didn't want it to be mixed up with the capability to install packaged apps (`packagedWebApps`). Any alternate suggestion ?

Comment (ngokevin) [inline core/capabilities.js]: Hmm, I don't think we'll get confused with using ```packaged``` or ```iframed``` since we throw those terms around a lot.

**Pull request merged**